Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntitySortComponent.cpp

```diff
@@ -51,6 +57,12 @@ namespace AzToolsFramework
                         ;
                 }
             }
+
+            AZ::JsonRegistrationContext* jsonRegistration = azrtti_cast<AZ::JsonRegistrationContext*>(context);
+            if (jsonRegistration)
+            {
+                jsonRegistration->Serializer<JsonEditorEntitySortComponentSerializer>()->HandlesType<EditorEntitySortComponent>();
```

Comment:
Potential derisk: we can not register this serializer and use the base serialization instead, but I'll have to fix up Activate to rely on the sort entry order and do some spot testing.